lib/kubeclient/common.rb

```diff
@@ -253,16 +254,18 @@ def watch_entities(resource_name, options = {})
       uri = @api_endpoint.merge("#{@api_endpoint.path}/#{@api_version}/#{path}")
 
       params = {}
+      format = options.delete(:raw) ? :text : :json
```

Comment:
I changed this to not use a delete when addressing other changes:  https://github.com/abonas/kubeclient/pull/262#issuecomment-324698152